src/lib/action.ts

```diff
@@ -206,4 +211,20 @@ export default class ActionHandler {
       this.setStyle(null, null, fill);
     }
   };
+
+  requestTeX = async (): Promise<void> => {
+    const text = prompt("Enter LaTeX source");
```

Comment:
probably this should become `window.prompt`; again, it's good to know where app becomes dependent on the browser